lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -72,6 +81,18 @@ public static function register_font_collection( $config ) {
 		return $new_collection;
 	}
 
+	/**
+	 * Unregisters a previously registered font collection.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $collection_id Font collection ID.
+	 * @return bool True if the font collection was unregistered successfully and false otherwise.
+	 */
+	public static function unregister_font_collection( $collection_id ) {
+		self::$unregistered_collection_ids[] = $collection_id;
+	}
```

Comment:
This feels like a weird pattern. Could we not just unset the value in `self::$collections`?

Reply:
Example 

https://github.com/WordPress/wordpress-develop/blob/ce8aed4698117fd0f8c362825e62d01586c8799d/src/wp-includes/class-wp-block-patterns-registry.php#L130-L153

Reply:
Thank you for the example. Very useful. I implemented that in the latest commits.